.github/workflows/release.yml

```
@@ -4,59 +4,234 @@ on:
+      - name: Download VSIX artifact
+        uses: actions/download-artifact@v4
+        with:
+          name: vsix
+
+      - name: Resolve VSIX path
+        run: |
+          set -euo pipefail
+          echo "VSIX_PATH=$(ls -1 *.vsix | head -n 1)" >> "$GITHUB_ENV"
+
       - name: Upload VSIX to GitHub Release
         uses: softprops/action-gh-release@v1
```

actionlint (1.7.9)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)